src/components/CurrentUser/CurrentUser.jsx

```diff
@@ -1,12 +1,64 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import './CurrentUser.scss';
 
-export const CurrentUser = () => (
-  <div className="CurrentUser">
-    <h2 className="CurrentUser__title"><span>Selected user: 2</span></h2>
+export class CurrentUser extends React.Component {
+  state = {
+    user: '',
```

Comment:
user its an object, right? then why here is a string

Reply:
I thought we need to set any empty value by default, but I'll fix it

Reply:
the value of state should never change his type